packages/material-ui/src/Theme/MaterialUIContext.tsx

```diff
@@ -1,92 +1,130 @@
-import React from 'react';
-import MaterialUIContextProps from './MaterialUIContextProps';
+import React, { ComponentType } from 'react';
+import { BoxProps, ButtonProps } from '@material-ui/core';
```

Comment:
Won't this break with v5?

Reply:
Doesn't break on my end for our v5 project. I am consuming the code via a forked publish at: https://www.npmjs.com/package/@rollercodester/material-ui